at_common_flutter/test/at_common_flutter_test.dart

```diff
@@ -1,5 +1,26 @@
+import 'dart:io';
+
+import 'package:at_common_flutter/at_common_flutter.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
+
+  test("get_width_ratio", () {
+    SizeConfig().screenWidth = 1100;
+    var res = SizeConfig().getWidthRatio(20);
+    expect(res, 20);
+  });
+
+  test("get_height_ratio", () {
+    SizeConfig().screenWidth = 1100;
+    var res = SizeConfig().getHeightRatio(20);
```

Comment:
Do we have to set screenHeight to get heightRation ?

Reply:
Screen width is required inside the function. The later calculation has to be ignored as we cannot mock Platform.
This is causing a lot of blockers in writing testcases since a lot of static methods and singleton classes are used.


<img width="1005" alt="Screenshot 2022-08-29 at 8 47 50 PM" src="https://user-images.githubusercontent.com/74557219/187235945-d2d42e53-d843-4094-addc-a58ad7b58c5f.png">